projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+
+***Type***
+: `string`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "authentication": {
+		        "jira": {
+			        "apiToken": "MyLittleJiraSecret"
+		        }
+	        }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_AUTHENTICATION_JIRA_APITOKEN="MyLittleJiraSecret"
+        ```
+
+<hr/>
+
+## `enabled`
+
+Enables or disables the plugin entirely.
+If set to `#!java false`, the plugin will be skipped completely and no results will be uploaded.
+
+***Environment variable***
+: `XRAY_ENABLED`
+
+***Type***
+: `boolean`
+: Accepted strings (case-insensitive):
+
+    - `#!json "true"`
+      `#!json "1"`
+      `#!json "y"`
+
+    - `#!json "false"`
+      `#!json "0"`
+      `#!json "n"`
+
+***Default***
+: `#!json false`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "enabled": "y"
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_ENABLED="y"
+        ```
+
+<hr/>
+
+## `projectKey`
+
+The Jira key of the project to upload results to.
+It is used for identification of relevant test cases, meaning that only test cases with an `#!java @XrayTest` annotation containing the project key will be tracked and eventually uploaded to Xray.
+
+***Environment variable***
+: `XRAY_PROJECTKEY`
+
+***Type***
+: `string`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "projectKey": "PRJ"
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_PROJECTKEY="PRJ"
+        ```
+
+<hr/>
+
+## `resultsUpload`
+
+These settings determine the plugin's behaviour regarding the results upload.
+
+### `assignee`
+
+The Jira user to assign created test executions to.
+The following values should be provided here:
+
+- :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } The account ID, usually of the form `61f8f589e688d6007068a792`.
+  You can retrieve account IDs by exporting an issue as XML where the user is visible (reporter, assignee, ...). The XML element containing the account ID will then look something like this:
+  ```xml
+  <assignee accountid="61f8f589e688d6007068a792">John Miller</assignee>
+  ```
+- :fontawesome-solid-server:{ title="Xray server" .xray-icon } The username used for login, for example `miller_j`.
+  You can retrieve usernames of other users by exporting an issue as XML where the user is visible (reporter, assignee, ...). The XML element will then look something like this:
+  ```xml
+  <assignee>miller_j</assignee>
+  ```
+
+!!! note
+    If the configured assignee is `#!json null`, the test execution issues will not be assigned to anyone.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_ASSIGNEE`
+
+***Type***
+: `string`
+
+***Default***
+: `#!json null`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "resultsUpload": {
+            "assignee": "61f8f589e688d6007068a792"
```

Comment:
code indention is not correct

Reply:
Fixed in bdf5080ab870aac828d9e70d10cab3594ba08cd2.